db/migrate/20220706181020_createlikes.rb

```diff
@@ -0,0 +1,5 @@
+class Createlikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes, &:timestamps
+  end
+end
```

Comment:
- [ ] You are missing the association between the likes, the post table and the user table. It can be done here with `t.references :author, foreign_key: { to_table: 'users' }` # this will create author_id and a corresponding index  
`t.references :post, foreign_key: true`
Or you can also add column and index in separate commands